tests/phpunit/tests/media.php

```diff
@@ -78,9 +78,10 @@ public static function tear_down_after_class() {
 	/**
 	 * Ensures that the static content media count, fetchpriority element flag and related filter are reset between tests.
 	 */
-	public function set_up() {
-		parent::set_up();
+	public function tear_down() {
+		parent::tear_down();
 
+		$this->set_main_query( new WP_Query() );
```

Comment:
Technically speaking, it would be more appropriate here to set the main query to the current value of `$wp_query`, which at this point will have already been reset to a fresh `WP_Query` instance by the test base class's `tear_down()` method.

Doesn't matter for the results of these tests though, so not a blocker.
```suggestion
		$this->set_main_query( $GLOBALS['wp_query'] );
```

Reply:
If https://github.com/WordPress/wordpress-develop/pull/4823 gets committed this should be removed anyway, but happy to make this change in the mean time.